ros_installation.md

```diff
@@ -0,0 +1,197 @@
+repositories which host Gz simulator and Gz libraries: one is `packages.ros.org` and the
+other is `packages.osrfoundation.org`. At the time of writing:
+
+ * ***packages.ros.org***
+   * ROS1 Noetic: Gz Citadel
+   * ROS2 Foxy: Gz Citadel
+   * ROS2 Galactic: Gz Edifice
+   * ROS2 Humble: Gz Fortress
+   * ROS2 Rolling: Gz Fortress (changing frequently)
+
+ * ***packages.osrfoundation.org***
+   * Gz Citadel
+   * Gz Edifice
+   * Gz Fortress
+   * Gz Garden
+
+This means that including the osrfoundation repository is not strictly needed
+to get the Gazebo binary package in Ubuntu. It can be installed from the ROS
+repository.
+
+## Installing the default Gazebo version for a ROS distribution using binary installations
+
+For users that need to run a specific version of ROS and want to use all
+the gazebo ROS related packages out-of-the-box, this is the recommended
+section:
+
+### ROS2 Humble and ROS2 Rolling
+ROS 2 Humble and ROS 2 Rolling support officially the Gazebo Fortress version.
+For a fully-integrated ROS system, Fortress is the recommened version. The way
+to proceed is just to use the ROS repository and do ***not*** use the
+osrfoundation repository.
+
+ROS packages `ros-humble-ros-gz` and `ros-rolling-ros-gz` will be available
+, these packages automatically install the necessary dependencies for Gazebo Fortress.
+
+### ROS 2 Galactic 
+ROS 2 Galactic officially supports Gazebo Edifice
+For a fully-integrated ROS system, Edifice is the recommened
+version. The way to proceed is just to use the ROS repository and do ***not***
+use the osrfoundation repository.
+
+ROS packages `ros-galactic-ros-gz` will be available pulling the necessary
+Gazebo Edifice dependencies.
+
+### ROS 2 Foxy
+ROS 2 Foxy supports officially the Gazebo Citadel
+version. For a fully-integrated ROS system, Citadel is the recommened
+version. The way to proceed is just to use the ROS repository and do ***not***
+use the osrfoundation repository.
+
+ROS packages `ros-foxy-ros-gz` will be available pulling the necessary
+Gazebo Citadel dependencies.
+
+### ROS 1 Noetic
+ROS Noetic officially supports Gazebo Citadel
+version. For a fully-integrated ROS system, Citadel is the recommened
+version. The way to proceed is just to use the ROS repository and do ***not***
+use the osrfoundation repository.
+
+ROS packages `ros-noetic-ros-gz` will be available pulling the necessary
+Gazebo Citadel dependencies.
+
+## Using an specific Gazebo version with ROS 2
+***Warning!:*** Using this option, you won't be able to use any ROS Ubuntu package
+related to Gz libraries from ROS deb repository.
+
+Selecting a different release of Gz than the one officially supported in each
+ROS distribution will require that you manually compile the [`ros_gz
+bridge`](https://github.com/gazebosim/ros_gz) from source. Same applies for
+other ROS packages using Gz libraries that could also require code changes in
```

Comment:
"changes in ..." is there something missing there?

Reply:
Probably I meant an "if" [a828de5](https://github.com/gazebosim/docs/pull/339/commits/a828de56d9c787b7c2ac52daea497a708453edef)